### DISCUSSION TITLE
Displaying images for icons

### DISCUSSION THREAD

**Discussion by corpooo** [FelixKratz/SketchyBar#85] [Ideas]
Hi!

Once again, l love the progress of this project. Just saw the 2.0 update and I can't wait to update my setup!

I am wondering if it would be possible to display images/icons in the bar.  

One possible use case:

Displaying the icon of the focused app

Thank you again for this project and keep up the good job! 😄

**Comment by es183923** [comment]
check #69

**Comment by FelixKratz** [comment]
Yes, this is possible and I plan to include it soon. I want to overhaul the item alias system while doing this, so it might still take some weeks, but it is on my todo list.

If you want to have the focussed app icon functionality *now* you can always look for an app for the default mac menu bar and alias it into sketchybar. For example: [Menu Bar Dock](https://github.com/EthanSK/Menu-Bar-Dock) can be configured to show the focussed app icon in the menu bar.
